backend/app/core/distributed_lock.py

```
@@ -252,6 +255,125 @@ def _watchdog() -> None:
+            try:
+                redis_url = getattr(settings, "CELERY_BROKER_URL", None) or getattr(
+                    settings, "REDIS_URL", None
+                )
+                if redis_url:
+                    self._async_redis_client = AsyncRedis.from_url(
+                        redis_url,
+                        decode_responses=True,
+                        socket_timeout=5,
+                        socket_connect_timeout=5,
+                    )
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)